File: src/main/java/app/askresume/global/config/jpa/AuditingConfig.java
package app.askresume.global.config.jpa;

import org.springframework.context.annotation.Bean;
import org.springframework.context.annotation.Configuration;
import org.springframework.data.domain.AuditorAware;
import org.springframework.data.jpa.repository.config.EnableJpaAuditing;

@Configuration
@EnableJpaAuditing
public class AuditingConfig {

    @Bean
    public AuditorAware<String> auditorAware() {
        return new AuditorAwareImpl();
    }

}

File: src/main/kotlin/app/askresume/global/config/jpa/AuditingConfig.kt
package app.askresume.global.config.jpa

import org.springframework.context.annotation.Bean
import org.springframework.context.annotation.Configuration
import org.springframework.data.jpa.repository.config.EnableJpaAuditing


@Configuration
@EnableJpaAuditing
class AuditingConfig {

    @Bean
    fun auditorAware() = AuditorAwareImpl()

}